tools/azure-sdk-tools/devtools_testutils/proxy_startup.py

```diff
@@ -87,6 +87,17 @@ def check_availability() -> None:
         return 404
 
 
+def check_system_proxy_availability() -> None:
+    """Checks for SSL_CERT_DIR and REQUESTS_CA_BUNDLE environment variables."""
+    ssl_cert = "SSL_CERT_DIR"
+    ca_bundle = "REQUESTS_CA_BUNDLE"
+
+    if not os.environ.get(ssl_cert):
+        _LOGGER.error(f"Please check '{ssl_cert}' path")
+    if not os.environ.get(ca_bundle):
+        _LOGGER.error(f"Please check '{ca_bundle}' path")
```

Comment:
Nitpicking to make the error message a bit more exact:
```suggestion
    if not os.environ.get(ssl_cert):
        _LOGGER.error(f"Please ensure the '{ssl_cert}' environment variable is correctly set in your test environment")
    if not os.environ.get(ca_bundle):
        _LOGGER.error(f"Please ensure the '{ca_bundle}' environment variable is correctly set in your test environment")
```

Reply:
Never mind, it's ok! Your suggestion is much better.